src/goto-programs/add_race_assertions.cpp

```diff
@@ -181,7 +189,8 @@ void add_race_assertions(
   w_guardst w_guards(context);
 
   Forall_goto_functions(f_it, goto_functions)
-    add_race_assertions(value_sets, context, f_it->second.body, w_guards);
+    if(f_it->first != goto_functions.main_id())
```

Comment:
Sorry for the late reply to this PR, but what does this line do? Are we ignoring data races in main?

Reply:
Yes, I don't think the assignment in `_ESBMC_main` needs to be considered. They are all initialization operations and do not participate in thread interleaving.